src/main/java/project/model/dto/smarttender/Tender.java

```diff
@@ -283,29 +284,26 @@ public void setAdditionalProperty(String name, Object value) {
     }
 
     @Override
-    public String toString() {
```

Comment:
Do I understand this change correctly, and Tender.class does not have toString() realization overriding for now?